src/irc/dcc/dcc-get.c

```diff
@@ -580,6 +588,7 @@ static void cmd_dcc_get(const char *data)
 
 void dcc_get_init(void)
 {
+	dcc_get_recv_buffer = NULL;
```

Comment:
Could move this = NULL to the initialization (although i think being .bss stuff this isn't needed at all)